misc/check_pr_title.py

```diff
@@ -73,9 +73,13 @@ def get_old_ver():
         ver = semver.VersionInfo.parse(ver)
     except ValueError:
         exit(f'Invalid SemVer version: {ver}')
-    oldver = get_old_ver()
-    if ver not in [oldver.bump_minor(), oldver.bump_patch()]:
-        exit(f'Version bump incorrect: {oldver} -> {ver}')
+    try:
+        oldver = get_old_ver()
+    except ValueError:
```

Comment:
```suggestion
    except git.exc.GitCommandError:
```
[workflow] Fix PR title checker "fix fatal: bad revision 'master'" in release PR